sirepo/pkcli/jupyterhublogin.py

```diff
@@ -58,9 +60,21 @@ def maybe_create_sirepo_user(qcall, email, display_name):
         with qcall.auth.logged_in_user_set(u, method=sirepo.auth.METHOD_EMAIL):
             n = sirepo.sim_api.jupyterhublogin.create_user(
                 qcall=qcall,
-                check_dir=True,
             )
             qcall.auth_db.model("UserRole").add_roles(
                 roles=[sirepo.auth_role.for_sim_type("jupyterhublogin")],
             )
         return PKDict(email=email, jupyterhub_user_name=n)
+
+
+def old_user_dirs():
+    """Outputs commands to delete dirs for Jupyter users that are not in database."""
+    with sirepo.quest.start() as qcall:
+        m = ""
+        r = qcall.auth_db.model("JupyterhubUser").search_all_for_column("user_name")
```

Comment:
I think better to query the db each time. I would be curious which is faster but iterating over r for each user dir is not very performant. Likewise, if we had more users than we do it could be too much to pull them all into memory. 

So, the code below would become
```
if not qcall.auth_db.model("JupyterhubUser").unchecked_search_by(user_name=d.basename)
```

Reply:
[ad6298a](https://github.com/radiasoft/sirepo/pull/7027/commits/ad6298ae14d580069dd27cda09abc4bfad79f545)